proposed/phpdoc.md

```diff
@@ -1220,18 +1220,19 @@ The @method allows a class to know which 'magic' methods are callable.
 
 #### Syntax
 
-    @method [return type] [name]([type] [parameter], [...]) [description]
+    @method <static|> [name]([type] [parameter], [...])[: return type] [description]
```

Comment:
`<static|>` seems not correct according to [ABNF](https://tools.ietf.org/html/rfc5234). Dunno ABNF in all details, but `[static]` looks more appropriate.